src/bin/typescript-coverage-report.ts

```diff
@@ -75,7 +74,17 @@ program
   )
   .parse(argvWithVersion(process.argv));
 
-const options = getOptions(program);
```

Comment:
I'm ok to remove the `getOptions` function if you want. The limit of `commander` is that unfortunately doesn't give you the `defaultValue` when you try to access it.